sw/host/opentitantool/src/command/transport.rs

```diff
@@ -32,6 +35,42 @@ impl CommandDispatch for TransportInit {
     }
 }
 
+/// Updates the firmware of the debugger/transport.  If no argument is given, opentitantool will
+/// fetch an appropriate firmware from a public archive (NOT IMPLEMENTED YET).  For instructions
+/// on how to build HyperDebug firmware locally, see
+/// https://docs.google.com/document/d/1ZEH7L5j9-wMw4tkW28-xt6JU5B6hTX0RdZD4h4OZzDo .
```

Comment:
I would prefer not to add network fetching of the firmware image to opentitantool itself.

The opentitan repo should have the correct firmware as a dependency and we can add a bazel rule that performs the update.  The help text can, of course, tell users where to download the images should they want to perform the update operation manually.

Reply:
Yes, I also abandoned the idea of downloading images from opentitantool itself.  I should update the comment, but let's settle on the approach first.